eZ/Publish/Core/MVC/Symfony/MVCEvents.php

```diff
@@ -85,4 +85,14 @@ final class MVCEvents
      * The event listener method receives a eZ\Publish\Core\MVC\Symfony\Event\ContentCacheClearEvent instance.
      */
     const CACHE_CLEAR_CONTENT = 'ezpublish.cache_clear.content';
+
+    /**
+     * The CONSOLE_INIT event allows you to attach listeners before any command is
+     * loaded by the console. It also allows you react to global arguments before commands are run.
+     *
+     * @Event("eZ\Publish\Core\MVC\Symfony\Event\ConsoleInitEvent")
+     *
+     * @internal For internal use by SiteAccess system on console, might move to symfony in some form in the future.
```

Comment:
```suggestion
     * @internal For internal use by SiteAccess system on console.
```

Removed part is also mentioned in https://github.com/ezsystems/ezpublish-kernel/pull/2687/files#diff-8d322be03e79835d42b3034a21073f8fR46 😉

Reply:
Ahhh, fun destroyer :P